### ISSUE TITLE
NPC patrol paths are not shown on minimap.

### ISSUE THREAD

**Issue by Cabro** [Questie/Questie#4954]
![image](https://github.com/Questie/Questie/assets/1155017/7985af25-2735-42c0-bfd0-074b4d28cf00)

8.4.2

**Comment by Dyaxler** [comment]
Have they ever been? I don't recall them being on the Minimap? I didn't write this part of Questie so I don't have a clue.

**Comment by Cabro** [comment]
I've never seen them either.

**Comment by Dyaxler** [comment]
I'm not even sure it's possible?

**Comment by MyTechnoHunter** [comment]
it would be a bit of a nightmare honestly, scaling and such being what it is.. -shudder-
also would take up screen real estate on a crowed, tiny map..

**Comment by Cabro** [comment]
SilverDragon can draw the paths on minimap

**Comment by MyTechnoHunter** [comment]
do we want to?
curious...

**Comment by Dyaxler** [comment]
If done right... maybe. But who would actually notice the path? Personally, I think I'd rather open the map to see the ENTIRE route not see a small piece on the Minimap. Some quest mobs have a LONG path...

**Comment by Cabro** [comment]
Just seeing that there is a path connected to the NPC will clear up the "?" that pop in your head when you see you are at the turn in location, but there's no NPC :-s

**Comment by Dyaxler** [comment]
True. That is somewhat annoying when we paint the finisher in a specific location and it's not there... especially when the Blizzard UI already paints the finisher and updates its location in real time when you get close enough to see him on the Minimap. I for one wouldn't mind a check... `if finisher has a path don't paint icon`

**Comment by Dyaxler** [comment]
At least not the finisher on the minimap... maybe keep the world map icon and indicate in the tooltip that he's along the painted path? Do we do that already? I don't remember off hand.

**Comment by Cabro** [comment]
On map the finisher is painted in the spawn coords + patrol path. These are linked.
For minimap its just the finisher in the spawn coords.

**Comment by Gogo1951** [comment]
Hey @Cabro ,

From a billion years ago, but Aero called out that this was possible (but not built) but we had decided against doing it because it added too much clutter to the minimap in trial, and you basically had to zoom out to see the full map anyway to see a reasonable path.

It just looked really messy. Possible we could have gotten the UX better, but it kind of made the mimimap look like there was ugly red graffiti on it without giving users a lot of useful data.

**Comment by Cabro** [comment]
I dont know what to say about that, there are other addons that draw lines on minimap and it's not cluttered. Probably too many useless paths when you tested it?
